quantum/nodes/shors/shors.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const util = require('util');
+const snippets = require('../../snippets');
+const errors = require('../../errors');
+const shell = require('../../python').PythonShell;
+
+module.exports = function(RED) {
+  function ShorsNode(config) {
+    RED.nodes.createNode(this, config);
+    this.name = config.name;
+    this.integer = config.integer;
+    const node = this;
+
+    this.on('input', async function(msg, send, done) {
+      const params = node.integer;
```

Comment:
I agree with @KhesimReid and should be coming from other nodes, rather than specified on configuration screen. So I suggest assigning value to params from msg payload.

Reply:
I agree with this, the value should be passed in `msg.payload`